### DISCUSSION TITLE
WooCommerce Custom Payment Integration

### DISCUSSION THREAD

**Discussion by dineshwar21** [woocommerce/woocommerce#51957] [Checkout Flow]
I am integrating a custom checkout in WooCommerce's block checkout. When a customer selects a field in my payment method, I need to refresh the checkout fields to recalculate the fees based on the selected method. How can I implement this behavior in the new block checkout?

How to reload the checkout using reactjs

**Comment by mikejolley** [comment]
There are events fired when payment method are changed, and this can be used to trigger the refresh.

See here https://github.com/woocommerce/woocommerce/issues/48497 and the example here https://gist.github.com/mikejolley/f185f73f0046353f05773843bd153a9a

Thanks
